master/buildbot/schedulers/basic.py

```diff
@@ -46,7 +46,8 @@ class BaseBasicScheduler(base.BaseScheduler):
     class NotSet:
         pass
 
-    def __init__(self, name, shouldntBeSet=NotSet, treeStableTimer=None,
+    def __init__(self, name, shouldntBeSet=NotSet, 
+                 treeStableTimer=None, coolDownTimer=None,
```

Comment:
I hope not, this is pretty risky to do. Isn't it a flake/pylint check for that? Maybe E1124 "Parameter %r passed as both positional and keyword argument"